initialization.py

```diff
@@ -4,7 +4,7 @@
 import xml.etree.ElementTree as ET
 import json
 
-f = open("tag-init.json", "r")
+f = open("./example-tags/tag-init.json", "r")
```

Comment:
Why create an own folder for the single json? Just move it into the "example-folder-structure/". Or better: Create a new parent folder for initialization stuff called smth like "initial-setup". In there move the "example-folder-structure/" and the "tag-init.json" file.

Reply:
done